README.md

```diff
@@ -19,7 +19,15 @@ $ "$GOPATH/bin/wuzz" --help
 ### Commands
 
 Keybinding                              | Description
-----------------------------------------|------------------------------------------------------------
+----------------------------------------|------------------------------------------------------------	
+<kbd>F2</kbd>                           | URL
+<kbd>F3</kbd>                           | Query parameters
+<kbd>F4</kbd>                           | HTTP method
+<kbd>F5</kbd>                           | Request body
+<kbd>F6</kbd>                           | Headers
+<kbd>F7</kbd>                           | Search
+<kbd>F8</kbd>                           | Response headers
+<kbd>F9</kbd>                           | Response body
```

Comment:
Perhaps, it would be better to place this window navigation block below the other keys. To emphasize the most used features like sending a request.